x/ccv/staking/module.go

```diff
@@ -0,0 +1,53 @@
+package staking
+
+import (
+	"encoding/json"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+	"github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	"github.com/cosmos/cosmos-sdk/x/staking/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+var (
+	_ module.AppModule           = AppModule{}
+	_ module.AppModuleBasic      = AppModuleBasic{}
+	_ module.AppModuleSimulation = AppModule{}
+)
+
+// AppModule embeds the Cosmos SDK's x/staking AppModuleBasic.
+type AppModuleBasic struct {
+	staking.AppModuleBasic
+}
+
+// AppModule embeds the Cosmos SDK's x/staking AppModule where we only override
+// specific methods.
+type AppModule struct {
+	// embed the Cosmos SDK's x/staking AppModule
+	staking.AppModule
+}
+
+// NewAppModule creates a new AppModule object using the native x/staking module
+// AppModule constructor.
+func NewAppModule(cdc codec.Codec, keeper keeper.Keeper, ak types.AccountKeeper, bk types.BankKeeper) AppModule {
+	stakingAppMod := staking.NewAppModule(cdc, keeper, ak, bk)
+	return AppModule{stakingAppMod}
+}
+
+// InitGenesis performs a no-op in that it returns no validator set updates.
+// Recall, validators are tracked via the consumer chain's x/cvv/consumer module
+// and so this module is not responsible for returning the initial validator set.
+func (am AppModule) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, data json.RawMessage) []abci.ValidatorUpdate {
```

Comment:
I think `InitGenesis` should still execute the logic from the original staking module (x/staking), since that initializes the internal state based on the genesis state. Once the logic is executed, the function should indeed return an empty validator set.